### DISCUSSION TITLE
Aribyte - Nginx 502 Bad Gateway error

### DISCUSSION THREAD

**Discussion by NChittimalla** [airbytehq/airbyte#45429] [Deployment]
I can see the deployment file for airbyte-server, but the Helm chart is not creating a pod for it. As a result, I’m unable to connect to the airbyte-web-app for the backend service and am encountering an Nginx 502 Bad Gateway error. How can I resolve this? Am I missing something in the configuration?

In my local environment, everything is working fine, and the application runs on localhost

Please find the attachments
![nginx-error](https://github.com/user-attachments/assets/81553ed3-9b3b-4d38-8d52-047f5cade022)

![pods](https://github.com/user-attachments/assets/2d926d5e-dfcb-41d0-815a-19e779f83768)

**Comment by marcosmarxm** [comment]
Can you provide what steps, what version of abctl, are you using custom nginx or values.yaml file?

**Reply by NChittimalla** [reply]
Hi @marcosmarxm Thanks for the response,
Im using values.yaml file itself, please find the attachment of it.
[values.yaml.pdf](https://github.com/user-attachments/files/16988845/values.yaml.pdf)

**Reply by NChittimalla** [reply]
Actually Iam getting one error, due to this i think server pod is not creating, but iam unable to remove this error.

* Deployment.apps "airbyte-workload-launcher" is invalid: [spec.template.spec.containers[0].env[63].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[64].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
        * Deployment.apps "airbyte-server" is invalid: [spec.template.spec.containers[0].env[41].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[42].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]
        * Deployment.apps "airbyte-worker" is invalid: [spec.template.spec.containers[0].env[48].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*'), spec.template.spec.containers[0].env[49].valueFrom.secretKeyRef.name: Invalid value: "": a lowercase RFC 1123 subdomain must consist of lower case alphanumeric characters, '-' or '.', and must start and end with an alphanumeric character (e.g. 'example.com', regex used for validation is '[a-z0-9]([-a-z0-9]*[a-z0-9])?(\.[a-z0-9]([-a-z0-9]*[a-z0-9])?)*')]

**Comment by NChittimalla** [comment]
https://github.com/airbytehq/airbyte/discussions/45386
